recipes/hmntrimmer/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+cxx_compiler_version:  # [unix]
```

Comment:
Can you add a comment here on the rationale on the version override? That will help with long-term maintenance when this is updated in the future.

Reply:
Yes, it's added ccaa5a9